README.md

```diff
@@ -271,6 +272,40 @@ logger2 = {"@loggers": "spacy.LookupLogger.v1", "substring": "pytorch"}
 | `cuda_mem_pool`   | `str` | One of the memory pool values specified in the PyTorch docs: `all`, `large_pool`, `small_pool` (default: `all`).                                                |
 | `cuda_mem_metric` | `str` | One of the memory metric values specified in the PyTorch docs: `current`, `peak`, `allocated`, `freed`. To log all metrics, use `all` instead (default: `all`). |
 
+## CupyLogger
+
+### Installation
+
+This logger requires `cupy` to be installed:
+
+```bash
+pip install cupy
+```
+
+### Usage
+
+Similar to `PyTorchLogger`, `spacy.CupyLogger.v1` is intended to be used with [ChainLogger](#chainlogger).
```

Comment:
Do you think it's helpful to call out the similarity with `PyTorchLogger`? Users might just only reading the `CupyLogger` section, independent from any other information on this page, and feel like they need to look up this explanation from somewhere else.